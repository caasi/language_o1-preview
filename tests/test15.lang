type Result = Success Number | Failure String
type MaybeResult = Just Result | Nothing

let res = Just (Success 100) in res