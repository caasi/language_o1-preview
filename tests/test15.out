Just (
  Success (
    100.000000
  )
)
